docs/configuration-validation.md

```diff
@@ -17,3 +17,17 @@ If you were presented with configuration error (or a warning, depending on `conf
 **Note**: In a warning mode (with `configValidationMode: warn` set in configuration) Framework commands are not blocked in any way, e.g. `sls deploy` will still attempt to deploy the service normally (still depending on the source of the warning, success of a deployment may vary)
 
 If you find this functionality problematic, you may also turn it off with `configValidationMode: off` setting.
+
+## Configuration
+
+### Where
+
+At the root of the `serverless.yml` file, add a `configValidationMode` key.
```

Comment:
Let's write it as:
`Add `configValidationMode` to your service configuration with one of the following values:`

The reason for this is you don't necessarily have to use yaml-based config